handler.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+const AWS = require('aws-sdk');
+const ccxt = require('ccxt');
+const axios = require('axios');
+
+module.exports.transaction = async (event) => {
+  var jobId = event.Records[0].Sns.Message;
+
+  var jobURL = `http://jobs.service.thinblock.io/jobs/${jobId}?look_up=_id`;
+  const { data: { actions: [ action ] } } = await axios.get(jobURL);
```

Comment:
@abdullahtariq91 We need to add this await code in try catch block i think.

Reply:
Also, you need to forEach and check if the action matches with the one currently executing, as we can have many actions. Get the `topic_arn` from the event and check against it.

Reply:
I'll add the `forEach` block. 👍